webrtc-stats.html

```diff
@@ -1554,9 +1554,17 @@ <h2>
               </dt>
               <dd>
                 <p>
+                  The purpose of the jitter buffer is to recombine RTP packets into frames (in the case of video)
+                  and have smooth playout. The model described here assumes that the samples or frames are 
+                  still compressed and have not yet been decoded.
                   It is the sum of the time, in seconds, each <a>audio sample</a> or video frame takes from
-                  the time it is received and to the time it exits the jitter buffer. This increases
-                  upon samples or frames exiting, having completed their time in the buffer (and
+                  the time it the first packet received by the jitter buffer (ingest timestamp) and to the 
+                  time it exits the jitter buffer (emit timestamp). 
+                  In the case of audio, several samples belong to the same RTP packet, hence they will have the same 
+                  ingest timestamp but different jitter buffer emit timestamps. 
+                  In the case of video, the frame is received over several RTP packets, hence the ingest timestamp
+                  is the earliest packet of the frame that entered the jitter buffer.
+                  This increases upon samples or frames exiting, having completed their time in the buffer (and
```

Comment:
Would it also be valid to count per-packet? So that even if a frame spans 3 different packets, this would yield 3 different emitted counts?

Reply:
the packets would have the same emit timestamp, it kinda depends on what we want to divide the number with? frames or packets?

I think frames.